doc/ref_model/chapters/chapter05.rst

```diff
@@ -169,6 +169,71 @@ Tracing                                Jaeger
 
 **Table 5-7a:** Service examples.
 
+Platform Services - Load Balancer Requirements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The table below specifies a set of requirements for the Monitoring service (aka monitoring sysstem).
+
++-------------+-----------------------------------------------------------------------+-------------------------------------------------------+
+| Reference   | Requirement                                                           | Notes                                                 |
++=============+=======================================================================+=======================================================+
+| pas.mon.001 | The Monitoring service must be able to collect data generated by or   | Capabilities to monitor applications, services,       |  
```

Comment:
I like the text in Notes as being specific.  "must be able to .. from any resource" seems to me much too strong.  For example, it will be very hard to expect any modern system to monitor some old legacy network elements.

Reply:
@walterkozlowski One mechanism of monitoring includes generating, for example, alerts from a logging system (passive alerting) or alerts generated by the monitored system.

Reply:
@pgoyal01, I understand.  But still "must" from any" seems terribly strong to me....  I will try to think of a better wording.